### ISSUE TITLE
Turbolinks config doesn't flush sprockets cache

### ISSUE THREAD

**Issue by ryanb** [wallaby-rails/wallaby-rails#171]
The `readyPrefix` function in `setup.js.erb` is dynamically generated depending on the turbolinks config. Unfortunately sprockets seems to cache this file and doesn't flush it until the file contents change. This means toggling the config will not update the asset cache.

Perhaps remove the dynamic generation in favor of detecting Turbolinks through JS.

**Comment by tian-im** [comment]
Yeah, the configuration is deprecated and it's detecting Turbolinks through JS instead.
